bin/daru-view

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env ruby
+
+require 'optparse'
+require 'rake'
+
+Dir.chdir File.expand_path('..',__dir__)
+
+parser = OptionParser.new do |opts|
+  opts.banner = "Usage: daru-view [options]"
+  # TODO add arguments to update only modules/adapters/a particular file
+
+  # Update google charts javascript dependent files, from latest Builds
+  # on the google developers website
+  opts.on('-g', '--update-googlecharts', 'Update googlecharts.js libraries') do
+    update(:googlecharts)
```

Comment:
create a folder `googlecharts_js`, if it is not present. Better place will be in rake task.